stock_virtual_available_adj/tests/test_product.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2020 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestProduct(common.TransactionCase):
+    """
+        This tests allows to check website sale available quantity for product.
+    """
+
+    def setUp(self):
+        super(TestProduct, self).setUp()
+
+        self.sale_order_line = self.env["sale.order.line"]
+        self.stock_location = self.env.ref("stock.stock_location_stock")
+        self.product_uom = self.env.ref("product.product_uom_unit").id
```

Comment:
```suggestion
        self.product_uom_id = self.env.ref("product.product_uom_unit").id
```
According to https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#variable-names.